prisma/schema.prisma

```diff
@@ -0,0 +1,24 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "mysql"
+  url      = env("DATABASE_URL")
+}
+
+model migrations {
+  id     Int      @id @default(autoincrement())
+  name   String
+  run_on DateTime
+}
+
+model parties {
```

Comment:
Are we going to add a `user` model as well?

Reply:
yep, will do that. Just need to be sure about the data model